code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm

```diff
@@ -20,7 +20,7 @@
 	speed = 1
 	melee_damage_lower = 15
 	melee_damage_upper = 18
-	damage_coeff = list(BRUTE = 1, BURN = 1.5, TOX = 1.5, CLONE = 0, STAMINA = 0, OXY = 1.5)
+	damage_coeff = list(BRUTE = 0.5, BURN = 1.5, TOX = 1.5, CLONE = 0, STAMINA = 0, OXY = 1)
```

Comment:
yes this is a stealth buff

Reply:
reasoning: gorillas 220hp is easily decimated by any kind of lynch mob attack, totally negating the scariness of having a gorilla rampaging on the station. i think this is balanced by the new lower odds of getting a gorilla.

also  if you've ever seen planet of the apes you know they can take a lot of bullets and i wanted to replicate that feeling of a gorilla eating entire magazines of bullets. note that lasers are 1.5 times more effective than base

Reply:
Then everyone would just use welders

Reply:
it was like this before this PR